### DISCUSSION TITLE
預知氣候災難不用靠阿南德！臺灣透過網路爬蟲程式，提升災害情資分析精準度

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#586] [科技新知]
從 2014 年起就擔任國家災害防救中心主任的陳宏宇，過去整合了學術研究理論模型以及颱風、洪水、地震等實際觀測數據，並透過網路爬蟲程式分析情資，為許多組織做出綜整分析，在因應氣候變遷打造韌性社會具有豐富實務經驗。
「我們無法避免天災、也很難預測天災的嚴重性，但我們可以超前部署，將傷害降至最低。」陳宏宇再一次強調，防災最重要的工作就是減災和避災，而要做到這一點，平常就要蒐集周全且完整的數據，並反覆進行災害情資的兵旗演練，瞭解各地方承受天災的能力，並找出可能受衝擊較大的地區，事前擬定防災計劃，才能從容面對可能的天災風險。

https://buzzorange.com/techorange/2022/04/27/extreme-climate/

**Comment by henrywen547** [comment]
現在也沒有任何證據指向星象變化會導致事件如何進展，到還不如相信科學數據的呈現。
利用網路爬蟲爬取大量資料後再進行分析，比較符合當代科學的作法。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期待未來的發展。

**Comment by cordydeng** [comment]
剛好搭上最近老師教的爬蟲，原來爬蟲除了搶票以外還能在這方面幫上忙

**Comment by Sazmafuji** [comment]
希望經過此堂課，也能寫出類似的東西

**Comment by sys511613** [comment]
用網路爬蟲也要注意假新聞辨識的能力，不然如果有太多惡意的假新聞，準確度可能會受到影響?

**Comment by jcpan821** [comment]
程式真的跟生活息息相關

**Comment by stan890425** [comment]
爬蟲也可以超前部署呢!

**Comment by alexthebigpp** [comment]
阿南德根本在亂講也不知道為甚麼這麼多人在理他
